### PR TITLE
revise JavaParser readme to reflect JSS inclusion

### PR DESCRIPTION
**Pull request by ftomassetti**
Minor changes to better describe all the features of JavaParser

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,17 +1,27 @@
...
 [![Coverage Status](https://coveralls.io/repos/javaparser/javaparser/badge.svg?branch=master&service=github)](https://coveralls.io/github/javaparser/javaparser?branch=master)
 [![Join the chat at https://gitter.im/javaparser/javaparser](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/javaparser/javaparser?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-## Dependency Management
+This project contains a set of libraries implementing a Java 1.0 - Java 9 Parser with AST generation and advanced features
+like:
+
+* comment attribution
+* lexical preservation
+* code generation
```
Comment (matozoid) [inline readme.md]: I'd be fine with moving this kind of descriptive content to javaparser.org.

File: readme.md [hunk 2]
```diff
@@ -1,17 +1,27 @@
...
+
+Our main site is at [JavaParser.org](http://javaparser.org)
 
-The project binaries are available in Maven Central.  Just add the following to your maven configuration or tailor to your own dependency management system.
+## Setup
+
+The project binaries are available in Maven Central. 
+
+We strongly advises users to adopt Maven, Gradle or another build system for their projects.
+
```
Comment (matozoid) [inline readme.md]: Maybe we can point to the quickstart projects here...

**Comment by matozoid** [comment]
We still have to move all the text from jss's old readme somewhere.

This PR is +1

File: readme.md (hunk 1 above)
Comment (ftomassetti) [reply readme.md]: I will remove the list but my problem is that seeing that JavaParser is just a parser could be reductive

File: readme.md (hunk 2 above)
Comment (ftomassetti) [reply readme.md]: Adding pointers

**Comment by ftomassetti** [comment]
Ok, I addressed your comments.
Regarding the content of the JSS readme we could put it on the website because the style of this readme is minimalistic and it probably does not fit here.

If there is one thing we could still is the cover of the book to be put in this Readme, near the link to the website.

File: readme.md (hunk 1 above)
Comment (matozoid) [reply readme.md]: I don't know what you mean :-)

**Comment by matozoid** [comment]
+1

**Pull request merged**